templates/_layouts/_header.html

```diff
@@ -15,9 +15,9 @@
       <ul class="p-navigation__items">
         <li class="p-navigation__item {% if path.startswith('/docs') and not path.startswith('/docs/examples') %}is-selected{% endif %}"><a class="p-navigation__link" href="/docs">Docs</a></li>
         <li class="p-navigation__item {% if '/docs/examples' in path %}is-selected{% endif %}"><a class="p-navigation__link" href="/docs/examples">Examples</a></li>
-        <li class="p-navigation__link{% if path == '/accessibility' %} is-selected{% endif %}"><a href="/accessibility">Accessibility</a></li>
-        <li class="p-navigation__link{% if path == '/browser-support' %} is-selected{% endif %}"><a href="/browser-support">Browser support</a></li>
-        <li class="p-navigation__link{% if path == '/contribute' %} is-selected{% endif %}"><a href="/contribute">Contribute</a></li>
+        <li class="p-navigation__item{% if path == '/accessibility' %} is-selected{% endif %}"><a class="p-navigation__link" href="/accessibility">Accessibility</a></li>
```

Comment:
Also I'm not sure why this is in this PR. It's fine if drive-by, but lets list it in the description (and link to fixed issues if we have them).

Reply:
There wasn't an issue for this, it was spotted when we initially switched the version to 3.0 and some deprecated styles were excluded. In any case, it's old markup that should have been updated, so I'll list it as an item in the description